hw9/OOP_examples.ts

```diff
@@ -0,0 +1,56 @@
+// Task
+// Provide examples applying OOP principles
+
+// Encapsulation
```

Comment:
Please add definition and explanation of each type

Reply:
We agreed to create a Readme file where each type is briefly described.
https://github.com/DariaBoris/aqa_repo/blob/master/README.md